derivation_method/super_method.py

```diff
@@ -930,7 +930,7 @@ def retrieve_json(self):
                     denominator: is_denominator,
                     required: is_required,
                     `all_ct`: should_all_ct
-                }
+                }, x_) YIELD rel
```

Comment:
does this work in both 4.x and 5.x?

Reply:
Yes, I don't know how was this missed but this test case was failing for 4.x earlier.